emote/nn/misc.py

```diff
@@ -0,0 +1,44 @@
+import torch.nn.functional as F
+
+from torch import Tensor, nn
+
+
+class FrozenBatchNorm1d(nn.BatchNorm1d):
```

Comment:
The issue and reason for this was that it was randomly flipping training/not training, right?

Reply:
Yes, this is correct

Reply:
The frozen_bn is only needed for the encoder and decoder models which are moved to erupt. Hence, I move this one also to erupt

Reply:
:+1: We can probably remove it completely, @spetravic and I looked at a similar issue caused by the ONNX exporter, and it now copies the network before setting the export state. We should maybe also make copies that are only used by the data generation loop that's always in infer mode.